wrappers/src/app_cs.cpp

```diff
@@ -40,16 +40,21 @@ using namespace app;
 using SharedSyncUser = std::shared_ptr<SyncUser>;
 using SharedSyncSession = std::shared_ptr<SyncSession>;
 
+using LogMessageCallbackT = void(void* managed_handler, realm_value_t message, util::Logger::Level level);
```

Comment:
I may be a silly question but, why do they have to be function pointers and not std::function-s? Is it because they have to have C-signature for exposure in shared_app_initialize?